lib/with_source_concatenation.rb

```diff
@@ -1,5 +0,0 @@
-module WithSourceConcatenation
-  def concatenate_source(content_src, extra_src)
```

Comment:
It was on the `ExpectationsRunner` too, but was removed when the compiler supported the incomplete AST generation.